GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -2220,6 +2174,7 @@ void FramebufferManagerCommon::ReadFramebufferToMemory(VirtualFramebuffer *vfb,
 		// We'll pseudo-blit framebuffers here to get a resized version of vfb.
 		if (gameUsesSequentialCopies_) {
 			// Ignore the x/y/etc., read the entire thing.
+			// TODO: What game did we need this for?
```

Comment:
IIRC, Grand Knights Historia was one of them but I think there were a few.  It was because some games would memcpy regions in strips, or many smaller regions from a larger framebuffer.

-[Unknown]